image_provisioner/playbooks/roles/delete-oldest/tasks/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+# Abort if target_directory is not set.
+- name: Checking for target_directory
+  fail:
+    msg: "Must provide the target_directory variable."
+  when: target_directory is not defined
```

Comment:
can you add the target_directory variable to the sample inventory files?

Reply:
I can add the variable to the files, but the way our Jenkins calls it is by copying an inventory outside of this repository so I will also add it to the inventory that Jenkins uses.